test/fixtures/google-gax-packaging-test-app/src/v1beta1/echo_client.js

```diff
@@ -197,8 +197,9 @@ class EchoClient {
       'wait',
       'pagedExpand',
     ];
+    this._innerCallPromise = {};
```

Comment:
Shall we have the same change to the generated code?

Reply:
sorry, you mean from the typescript microgenerator? Yes I'm making the change for the micro-gen to have the interface of async iterator, but this will not result in any braking changes for the current usage.

Reply:
Thank you!